src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -585,4 +611,17 @@ private function getWorkerExpiry()
 
         return $time->getTimestamp() - $now->getTimestamp();
     }
+
+    public function getDefaultSearchContext()
+    {
+        $context =  parent::getDefaultSearchContext();
+
+        $context->addField($this->buildJobStatusField()->setEmptyString(''));
```

Comment:
Fields should be added in `scaffoldSearchFields()`

Reply:
Done